### PR TITLE
29 implement add contact componenet rb dh

### PR DESCRIPTION
**Pull request by RajaaBoulassouak**
#### Description
- Implement JSX required for Create Contact Component in CreateContact.js file
- Add test for Create Contact component in CreateContact.test.js file
- Specify view height App.css 
- Specify body in App.js
- Add css for styling Create Contact page in CreateContact.css file (WIP)

#### Issues # and Action
- references #29 
----
#### TO DO / Noteworthy
- Syntax and labeling in CreateContact.js file is causing console.errors
- Notes input field doesn't render
----
#### Features
- [ ] New Feature
- [ ] Expand Feature
- [ ] Optimize/Improve Feature

----
#### Testing
- [ ] No Testing
- [ ] Some Testing
- [x] Sufficient Testing

- [ ] Left notes near functions that have not been tested
- [ ] Intentionally skipped / failing / empty tests as a reminder to add/improve these later

- [ ] Tests have been changed
- [ ] Existing tests are now failing / skipped

----
#### Deployment
- [ ] This is a complete feature, ready for deployment
- [x] This is a partial feature, NOT ready for deployment

----
#### API Changes
(please detail in description)
- [ ] New API
- [ ] Existing API, New Endpoint
- [ ] README.md is updated

----
#### Tech Changes
(please describe)
- [ ] New Functional Library
- [ ] New Testing Library
- [ ] New Debugging Tool
- [ ] README.md is updated

### CODE REVIEW

File: src/App/App.css [hunk 1]
```diff
@@ -1,5 +1,6 @@
 .app {
   background-color: #F2F3F9; 
+  height: 100vh;
 }
```
Comment (drhanciles) [inline src/App/App.css]: let's remove the `height` property here so we don't have an merge conflicts.

**Review by drhanciles — COMMENTED** [review]
Great work on this Rajaa! I have made one comment on something that I would like to have changed before we merge this branch in. Once you get it changed let me know and I'll merge this branch in. 
@RajaaBoulassouak

**Comment by RajaaBoulassouak** [comment]
Thank you, Derek. The height is now removed from App.js.

**Comment by drhanciles** [comment]
Merging this in now!

**Pull request merged**